### ISSUE TITLE
Shift and Trace scaling problems...

### ISSUE THREAD

**Issue by gab3d** [opentoonz/opentoonz#3797]
@shun-iwasawa 
I'm seeing a weird behavior when trying to scale reference drawings, especially the Following Drawing, and more noticeable even when using the path feature and trying to scale the Following Drawing...
It seems to be something related to the center used to scale.
Whatever it is, it turns the operation uncontrollable. 🤔 

To test I've used a vector level, with one Circle in each reference frame.

**Comment by gab3d** [comment]
@shun-iwasawa 
Sorry, any chance this could get looked at?
It's a really great tool and it's a pitty this problem makes it sometimes frustrating to use.

**Comment by shun-iwasawa** [comment]
@gab3d 
"A weird behavior" means an irregular wobbling behavior when dragging scale handle, right?
I think I could reproduce it at my end.

**Comment by gab3d** [comment]
@shun-iwasawa 
Sorry for not being more descriptive in my report.
I've just checked the Artifact in #3877 and unfortunately I still see the scaling issues, basically:
- Scale not using the transformation Center (rotation does use it).
- An unstable "jumpy" behavior when scaling (specially after having rotated the shape).

I've made a video depicting the issue.
Hope this helps...  


https://user-images.githubusercontent.com/18104091/114183423-0f4f8080-991a-11eb-814e-14ed47ad6690.mp4
